File: marcos006/programacao_basica/Questao17.java
import java.util.Scanner;

    public class Questao17 {
        public static void main(String[] args) {
            Scanner s;
            s = new Scanner(System.in);
        
        System.out.println("** Programa calculador do quanto foi poupado **");
        System.out.println("Digite a quantidade de moedas de 1 centavo: ");
        double u = s.nextDouble();
    
        System.out.println("Digite a quantidade de moedas de 5 centavos: ");
        double c = s.nextDouble();

        System.out.println("Digite a quantidade de moedas de 10 centavos: ");
        double d = s.nextDouble();
    
        System.out.println("Digite a quantidade de moedas de 25 centavos: ");
        double v = s.nextDouble();

        System.out.println("Digite a quantidade de moedas de 50 centavos: ");
        double cc = s.nextDouble();

        System.out.println("Digite a quantidade de moedas de 1 real: ");
        double r = s.nextDouble();

        double uu = u * 0.01;
        double co = c * 0.05;
        double dd = d * 0.1; 
        double vv = v * 0.25;
        double ccc = cc * 0.50;
        double rr = r * 1.0;

        double to = uu + co + dd + vv + ccc + rr;

        System.out.println("A quantidade total que pedrinho popou foi " + to + " R$");

    }
}

File: marcos006/programacao_basica/questao17.py
print("** Programa calculador do quanto foi economizado**")
u = float(input("Digite a quantidade de moedas de 1 centatavo: "))
c = float(input("Digite a quantidade de moedas de 5 centavos: ")) 
d = float(input("Digite a quantidade de moedas de 10 centavos: "))
v = float(input("Digite a quantidade de moedas de 25 centavos: "))
cc = float(input("Digite a quantidade de moedas de 50 centavos: "))
r = float(input("Digite a quantidade de moedas de 1 real: "))
uu = u * 0.01
co = c * 0.05
dd = d * 0.1
vv = v * 0.25
ccc = cc * 0.50
rr = r * 1.0

to = uu + co + dd + vv + ccc+ rr
print("A quantidade total que pedrinho poupou foi " + str(to))
